frontend/components/form/EmailChange.vue

```diff
@@ -0,0 +1,309 @@
+                            summary: t.value("common.toast.error.heading"),
+                            detail: t.value("wrong.password.toast.error"),
+                            life: 6000,
+                        });
+                    } else {
+                        toast.add({
+                            severity: "error",
+                            summary: t.value("common.toast.error.heading"),
+                            detail: t.value("common.error.unknown"),
+                            life: 6000,
+                        });
+                    }
+                },
+            });
+        }
+    }
+}
+</script>
+
+<template>
+    <div>
+        <Dialog
+            v-model:visible="isVisible"
+            modal
+            block-scroll
+            :auto-z-index="true"
+            :draggable="false"
+            class="z-50 flex w-2/3 flex-col rounded-lg bg-background font-nunito dark:bg-background-dark max-sm:collapse sm:rounded-xl lg:w-1/2 2xl:w-1/4"
+            :pt="{
+                root: {
+                    class: 'font-nunito bg-background dark:bg-background-dark z-10',
+                },
+                header: {
+                    class: 'flex justify-start pb-2 font-nunito bg-background dark:bg-background-dark',
+                },
+                title: {
+                    class: 'font-nunito text-4xl font-semibold',
+                },
+                content: {
+                    class: 'font-nunito bg-background dark:bg-background-dark px-0 sm:pl-5 sm:pr-12 h-full',
+                },
+                footer: { class: 'h-0' },
+                icons: {
+                    class: 'justify-end items-center w-fit pl-10',
+                },
+                closeButtonIcon: {
+                    class: 'z-30 self-center text-natural-500 hover:text-text dark:text-natural-400 dark:hover:text-text h-10 w-10 ',
+                },
+                mask: {
+                    class: 'max-sm:collapse',
+                },
+            }"
+            @hide="close"
+        >
+            <template #header>
+                <div class="flex w-[90%] items-center">
+                    <div class="h-0.5 w-5 bg-calypso-400" />
+                    <div
+                        class="flex-grow-5 mx-3 text-3xl text-text dark:text-natural-50"
+                    >
+                        <T key-name="dashboard.user.settings.email.change" />
+                    </div>
+                    <div class="h-0.5 flex-grow bg-calypso-400" />
+                </div>
+            </template>
+            <div class="pl-4">
+                <div
+                    class="-pt-4 overflow-hidden overflow-ellipsis text-sm text-natural-700 dark:text-natural-300"
+                >
+                    <T
+                        key-name="dashboard.user.settings.change.email.description"
+                    />
+                    {{ props.currEmail }}
+                </div>
```

Comment:
![image](https://github.com/user-attachments/assets/ca784666-a442-4c76-a17e-6f5d42184b39)
can u center the input fields and the buttons to the center (blue line shows the centre of the popup)